pylint_plugins/__init__.py

```diff
@@ -9,3 +10,4 @@ def register(linter):
     linter.register_checker(PrintFunction(linter))
     linter.register_checker(UnittestAssertRaises(linter))
     linter.register_checker(SetChecker(linter))
+    # linter.register_checker(StringChecker(linter))
```

Comment:
```suggestion
    linter.register_checker(StringChecker(linter))
```

@tsugumi-sys You can make this change to enable this rule in your PR.